File: sdks/swift/Sources/PuxbaySDK/Models/Branch.swift
import Foundation

public struct Branch: Codable {
    public let id: String
    public let name: String
    public let uniqueId: String
    public let address: String?
    public let phone: String?
    public let branchType: String
    public let currencyCode: String
    public let currencySymbol: String
    public let lowStockThreshold: Int
    public let createdAt: String
    public let updatedAt: String
}

File: sdks/rust/src/models/branch.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Serialize, Deserialize)]
pub struct Branch {
    pub id: String,
    pub name: String,
    pub unique_id: String,
    pub address: Option<String>,
    pub phone: Option<String>,
    pub branch_type: String,
    pub currency_code: String,
    pub currency_symbol: String,
    pub low_stock_threshold: i32,
    pub created_at: String,
    pub updated_at: String,
}
